Update package.json

anant2003 opened the pull request:
I have just change the name as covid19- tracker and version 0.2.0 of this project .

kavyanshpandey commented:
Hii, I added the label, but I can't merge it, kindly keep version name as it was. as it is the initial version.

anant2003 commented:
Now I have updated the initial version now you can merge it.

anant2003 commented:
now I have updated please check .

On Sat, Oct 2, 2021 at 4:45 AM Kavyansh Pandey ***@***.***>
wrote:

> Hii, I added the label, but I can't merge it, kindly keep version name as
> it was. as it is the initial version.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/kavyanshpandey/Covid19--Tracker/pull/14#issuecomment-932628522>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AREL32NV4FQ57NMHXKXYWPDUEY6KFANCNFSM5FEKZ7DA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>
